WordPress/WordPressTest/ViewRelated/Post/Controllers/PostListHelperTests.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import XCTest
+
+@testable import WordPress
+
+class PostListHelperTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testDateAndTimeFormatter() {
+        let date = Date.init(timeIntervalSince1970: 10)
+        let expectedDateString = "Dec 31, 1969 @ 7:00 PM"
```

Comment:
This test is failing for me. Probably due to timezone issues?

Reply:
I think I've resolved the timezone issue, but I'm still having problems visualizing the shape of the Post, and don't think I'm accessing the right value for the date/time of local changes. I _think_ the web date/time will be `post.dateModified` or `post.latest().dateModified` (shouldn't these be the same?), but I don't know how to get a `dateModified` for the local changes.

Reply:
I almost wonder if it would be beneficial to have a convenience property clearly labeled `localChanges`, that could be accessed? There are properties for `hasLocalChanges` and `isLocalDraft`, so but no clear way to grab those changes.